integration/experiment/launch_util.py

```diff
@@ -93,17 +94,20 @@ def launch_run(agent_conf, app_conf, run_id, output_dir, extra_cli_args,
     num_ranks = app_conf.get_rank_per_node() * num_nodes
     cpu_per_rank = app_conf.get_cpu_per_rank()
 
-    launcher = geopmpy.launcher.Factory().create(argv,
-                                                 num_node=num_nodes,
-                                                 num_rank=num_ranks,
-                                                 cpu_per_rank=cpu_per_rank)
-    launcher.run()
+    # Attempt to run if there are no valid report files for the current configuration
+    if not (os.path.isfile(report_path) and os.path.getsize(report_path) > 0):
```

Comment:
I thought this was managed by the creation of the machine file.  You cannot create a machine file if reports are present...

Reply:
That has not been my experience with the runs I did in December.  This was required.